### DISCUSSION TITLE
'x-html' is not rendering

### DISCUSSION THREAD

**Discussion by sobotkap10** [alpinejs/alpine#1578] [7. Bugs (archived)]
https://alpinejs.dev/directives/html

```
<div x-data="{ username: '<strong>calebporzio</strong>' }">
    Username: <span x-html="username"></span>
</div>

```

![image](https://user-images.githubusercontent.com/81413106/122732507-42948100-d274-11eb-90a9-a913922c3b38.png)

**Comment by ryangjchandler** [comment]
Related to #1576.
